include/boost/geometry/algorithms/discrete_hausdorff_distance.hpp

```diff
@@ -314,16 +314,15 @@ struct discrete_hausdorff_distance<default_strategy, false>
 
 
 /*!
-\brief Calculate discrete Hausdorff distance between two geometries (currently
-    works for LineString-LineString, MultiPoint-MultiPoint, Point-MultiPoint,
-    MultiLineString-MultiLineString) using specified strategy.
+\brief \brief_calc2{discrete Hausdorff distance, between} \brief_strategy
+\details \details_free_function{discrete_hausdorff_distance, discrete Hausdorff distance, between}.
 \ingroup discrete_hausdorff_distance
 \tparam Geometry1 \tparam_geometry
 \tparam Geometry2 \tparam_geometry
-\tparam Strategy A type fulfilling a DistanceStrategy concept
-\param geometry1 Input geometry
-\param geometry2 Input geometry
-\param strategy Distance strategy to be used to calculate Pt-Pt distance
+\tparam Strategy \tparam_strategy{Distance}
+\param geometry1 \param_geometry
+\param geometry2 \param_geometry
+\param strategy \param_strategy{point to point distance}
```

Comment:
Same as for Frechet. Now generates this doc:

![image](https://github.com/user-attachments/assets/2db3061a-6c2e-4346-8e0e-182f44a5d244)